### PR TITLE
Ah 2017 6 privacy botton

### PR DESCRIPTION
**Pull request by ahmedjenbaz**
fixed isse #431

### CODE REVIEW

File: euth/dashboard/forms.py [hunk 1]
```diff
@@ -114,7 +116,11 @@ class ProjectForm(forms.ModelForm):
     class Meta:
         model = project_models.Project
         fields = ['name', 'description', 'image', 'information', 'is_public',
-                  'result']
+        'result']
+        widgets = {'is_public': RadioSelect(choices=[
+            (True, 'Private'),
```
Comment (mkind) [inline euth/dashboard/forms.py]: Good work @ahmedjenbaz :smile: The translatable version ` _('Private')` might be better here.

File: euth/dashboard/forms.py [hunk 2]
```diff
@@ -115,6 +115,9 @@ class Meta:
         model = project_models.Project
         fields = ['name', 'description', 'image', 'information', 'is_public',
                   'result']
+        widgets = {'is_public': RadioSelect(choices=[
+                (True, _('Private')),
+                (False, _('Not Private'))])}
```
Comment (fuzzylogic2000) [inline euth/dashboard/forms.py]: 'Not Private' should be 'Public', and the booleans should be the other way round. (As the model field is called is_public, the field should be set true, when the project is public and false, when it is private.)

File: euth/dashboard/templates/euth_dashboard/includes/project_form_information_tab.html [hunk 3]
```diff
@@ -20,11 +20,9 @@ <h3 class="form-section-heading">{{ name }}</h3>
                 {{ field.label }} {% if field.field.required %}*{% endif %}
             </label>
             <p class='form-help-text'>{{ field.help_text }}</p>
-            {% if field|widget_type == 'checkboxinput' %}
+            {% if field|widget_type == 'radioselect' %}
             <label>
-                {% get_checkbox_label form.project field.name as checkboxlabel %}
```
Comment (fuzzylogic2000) [inline euth/dashboard/templates/euth_dashboard/includes/project_form_information_tab.html]: we still need to check if the tamplate tag is needed elsewhere.

File: euth/dashboard/templates/euth_dashboard/includes/project_form_information_tab.html [hunk 4]
```diff
@@ -20,11 +20,9 @@ <h3 class="form-section-heading">{{ name }}</h3>
                 {{ field.label }} {% if field.field.required %}*{% endif %}
             </label>
             <p class='form-help-text'>{{ field.help_text }}</p>
-            {% if field|widget_type == 'checkboxinput' %}
+            {% if field|widget_type == 'radioselect' %}
```
Comment (fuzzylogic2000) [inline euth/dashboard/templates/euth_dashboard/includes/project_form_information_tab.html]: do we still need this if for the styling? and the <label>?

File: euth/dashboard/templates/euth_dashboard/includes/project_form_information_tab.html [hunk 5]
```diff
@@ -20,14 +20,10 @@ <h3 class="form-section-heading">{{ name }}</h3>
                 {{ field.label }} {% if field.field.required %}*{% endif %}
             </label>
             <p class='form-help-text'>{{ field.help_text }}</p>
-            {% if field|widget_type == 'checkboxinput' %}
-            <label>
-                {% get_checkbox_label form.project field.name as checkboxlabel %}
```
Comment (fuzzylogic2000) [inline euth/dashboard/templates/euth_dashboard/includes/project_form_information_tab.html]: The assignment-tag get_checkbox_label and the function it uses can and should be deleted, as these are not used in any other template.

Comment (slomo) [inline euth/dashboard/templates/euth_dashboard/includes/project_form_information_tab.html]: I think this if in the tender logic (that checks the widget_type) could also be removed then.

**Pull request merged**